dev-libs/hyprlang/hyprlang-0.4.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Official implementation library for the hypr config language"
+HOMEPAGE="https://github.com/hyprwm/hyprlang"
+SRC_URI="https://github.com/hyprwm/${PN}/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BUILD_DIR="${S}/build"
+
+src_prepare() {
+	default
+
+	sed "s/\<lib\>/$(get_libdir)/" -i CMakeLists.txt hyprlang.pc.in
```

Comment:
Doesn't appear to be necessary https://github.com/hyprwm/hyprlang/commit/25da0804b00fffeee17463afd146711b4a05e77b